DESCRIPTION

```diff
@@ -22,7 +22,7 @@ BugReports: https://github.com/ropensci/riem/issues
 Imports:
     httr2,
     jsonlite (>= 0.9.19),
-    lubridate (>= 1.5.6),
+    lubridate (>= 1.9.0.9000),
```

Comment:
I now remember we need to add a field called Remotes recording where that version comes from.

```
Remotes:
    tidyverse/lubridate
```

If you use usethis simply run `usethis::use_dev_package("lubridate")`